packages/dapp/src/components/Auth/index.tsx

```diff
@@ -142,11 +179,11 @@ export class AuthRouter extends React.Component<RouteComponentProps> {
     });
   };
 
-  public handleOnAuthenticationContinue = (isNewUser: boolean): void => {
+  public handleOnAuthenticationContinue = (isNewUser: boolean, redirectUrl?: string): void => {
```

Comment:
I think we should set the account (and kick off action to subscribe to token balances) in redux here (and remove all the account stream subscriptions stuff in Main.tsx)

Reply:
@nickreynolds I like the idea of co-locating the account/balances dispatchers with the Auth logic, but this method only gets called when the user actually logins via the login UI. So we would still need a place to kickoff those account/balance subscriptions for users who already have a valid session.

For the sake of this PR being reviewable and discrete, I'll look at refactoring this code, including moving the account/balance subscriptions here, in an upcoming PR that'll land this week.